lib/src/view/game/game_list_detail_tile.dart

```diff
@@ -0,0 +1,306 @@
+import 'package:dartchess/dartchess.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:lichess_mobile/src/model/common/chess.dart';
+import 'package:lichess_mobile/src/model/game/archived_game.dart';
+import 'package:lichess_mobile/src/model/game/player.dart';
+import 'package:lichess_mobile/src/styles/lichess_colors.dart';
+import 'package:lichess_mobile/src/styles/lichess_icons.dart';
+import 'package:lichess_mobile/src/styles/styles.dart';
+import 'package:lichess_mobile/src/utils/l10n_context.dart';
+import 'package:lichess_mobile/src/view/game/game_common_widgets.dart';
+import 'package:lichess_mobile/src/view/game/game_list_tile.dart';
+import 'package:lichess_mobile/src/view/game/status_l10n.dart';
+import 'package:lichess_mobile/src/widgets/adaptive_bottom_sheet.dart';
+import 'package:lichess_mobile/src/widgets/board_thumbnail.dart';
+import 'package:lichess_mobile/src/widgets/user_full_name.dart';
+import 'package:timeago/timeago.dart' as timeago;
+
+/// A list tile that shows more detailed game info than [GameListTile].
+class GameListDetailTile extends StatelessWidget {
+  const GameListDetailTile({
+    required this.item,
+  });
+
+  final LightArchivedGameWithPov item;
+
+  Side get mySide => item.pov;
+
+  @override
+  Widget build(BuildContext context) {
+    final (game: game, pov: youAre) = item;
+    final me = youAre == Side.white ? game.white : game.black;
+
+    final customColors = Theme.of(context).extension<CustomColors>();
+
+    final dateStyle = TextStyle(
+      color: textShade(
+        context,
+        Styles.subtitleOpacity,
+      ),
+      fontSize: 12,
+    );
+
+    return GestureDetector(
```

Comment:
The problem with the gesture detector, is that there is no feedback that the item has been tapped.

It is better replaced with a `PlatformInkWell` to handle `onTap`, and the detector can be used only for the long press gesture.